Casks/1/115.rb

```diff
@@ -16,6 +16,8 @@
     end
   end
 
+  deprecate! date: "2024-07-25", because: :discontinued
```

Comment:
```suggestion
  disable! date: "2024-07-25", because: :no_longer_available
```

It seems like the client is no longer available for download and existing installs cannot log in to use the service. So we should disable the cask altogether.

Please also remove the `livecheck` block.